android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -170,6 +170,7 @@ public boolean accept(File dir, String filename) {
   public void cropImage(
       String uri,
       ReadableMap options,
+      boolean useExternalCache,
```

Comment:
As Mark said let's add this to options. Also let's have the reverse variable of `useExternalCache` (so it defaults to current behaviour). Otherwise this will be a breaking change for the library & will be less likely to be merged upstream

Reply:
maybe `internalCacheOnly` - to indicate this excludes use of other cashes

Reply:
@andrejborstnik  FIxed